sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/v2/example/ExampleResource.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi.api.v2.example;
+
+import io.grpc.Status;
+import io.grpc.StatusRuntimeException;
+import io.smallrye.mutiny.Uni;
+import io.stargate.proto.Schema;
+import io.stargate.sgv2.docsapi.api.common.RequestInfo;
+import io.stargate.sgv2.docsapi.api.common.exception.model.dto.ApiError;
+import io.stargate.sgv2.docsapi.api.v2.example.model.dto.KeyspaceExistsResponse;
+import io.stargate.sgv2.docsapi.config.constants.Constants;
+import io.stargate.sgv2.docsapi.grpc.GrpcClients;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.enums.ParameterIn;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameters;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.eclipse.microprofile.openapi.annotations.security.SecurityRequirement;
+import org.jboss.resteasy.reactive.RestResponse;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Example resource showcasing the OpenAPI v3 annotations and the reactive implementation.
+ */
+@Path("/api/v2/example")
+@Produces(MediaType.APPLICATION_JSON)
+@SecurityRequirement(name = Constants.OPEN_API_DEFAULT_SECURITY_SCHEME)
+public class ExampleResource {
+
+    @Inject
+    GrpcClients grpcClients;
+
+    @Inject
+    RequestInfo requestInfo;
+
+    @Operation(
```

Comment:
I did not extract OpenAPI definitions into the interface as the restapi v2, but this is definitely an option imo..